examples/qbv-example/build.gradle

```diff
@@ -0,0 +1,63 @@
+buildscript {
+	repositories {
+		mavenLocal()
+		mavenCentral()
+	}
+	dependencies {
+		classpath "com.marklogic:ml-gradle:4.6-SNAPSHOT"
+	}
+}
+
+plugins {
+	id "net.saliman.properties" version "1.5.1"
+	id "java"
+}
+
+apply plugin: "com.marklogic.ml-gradle"
+
+repositories {
+	mavenLocal()
+	mavenCentral()
+	maven { url "https://developer.marklogic.com/maven2/" }
+}
+
+configurations {
+	mlcp
+}
+
+dependencies {
+	mlcp 'com.marklogic:mlcp:10.0.9.5'
```

Comment:
Use latest version - 11.0.3 - particularly since the README mentions MarkLogic 11.